docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -4,189 +4,182 @@ description: How to choose and connect to a testnet
+Each Ethereum 2.0 validator needs access an Ethereum 1.0 client to onboard new validators.
+Validators make deposits into Ethereum 1.0, and existing Ethereum 2.0 validators must see these
+deposits and process them to enable the validators to join Ethereum 2.0.
 
-Teku does not contain a preset for testnet 5. The necessary configuration can be found in the [eth2-testnets repository on Github](https://github.com/eth2-clients/eth2-testnets/tree/master/lighthouse/testnet5).
+The steps to run an Ethereum 2.0 validator on a testnet are:
 
-## Enable an ETH1 Connection
+1. [Sync Besu to the Ethereum 1.0 network containing
+    the deposit contract](#sync-besu-to-the-ethereum-10-network).
 
-**Prerequisites**:
+1. [Load the validator deposit amount (plus gas) into
+    your Ethereum 1.0  deposit account](#load-the-deposit-account-with-eth).
 
-* [Install Hyperledger Besu](https://besu.hyperledger.org/en/latest/HowTo/Get-Started/Install-Binaries) Ethereum 1.0 client in addition to the Teku Ethereum 2.0 client.
+1. [Generate the validator key and send the deposit to the deposit
+    contract](#send-the-validator-deposit).
 
-The deposit contract for Schlesi and Topaz are deployed to the _Goerli_ ETH1 testing network. To enable an ETH1 connection on Teku, configure Hyperledger Besu to connect to the Goerli testnet and expose the RPC-HTTP APIs:
+1. [Start Teku with the validator key](#start-the-validator).
 
-```bash
-besu --network=goerli \
---rpc-http-enabled=true \
---rpc-http-port=8545 \
---rpc-http-api=ETH,NET,WEB3
-```
+### Sync Besu to the Ethereum 1.0 network
 
-Once you have a local Goerli node running with an activated RPC-HTTP API, you can connect Teku to it by enabling `--eth1-enabled`.
+The deposit contract for the testnets is located in the Goerli Ethereum 1.0 testnet.
+Configure Besu to [connect to Goerli] and expose the RPC-HTTP APIs.
 
 !!! example
 
-    ```
-    teku --network=schlesi --eth1-enabled --eth1-endpoint=localhost:8545
-    ```
-
-!!! tip
-
-    See also: [Hyperledger Besu - Run a node on Goerli testnet](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/#run-a-node-on-goerli-testnet)
-
-## Boostrap a new testnet
+    ```bash
+    besu --network=goerli --data-path=./goerli --rpc-http-enabled=true --rpc-http-port=8545 \
+    --rpc-http-api=ETH,NET,WEB3 --sync-mode=FAST --fast-sync-min-peers=2
+   ```
+  
+### Load the deposit account with ETH
 
-If you already have created a custom testnet or plan to do so, Teku allows you to pass different configurations to the beacon-chain node. A custom network requires a couple of things:
+You need an Ethereum 1.0 account that contains at least 32 ETH (plus gas). For the
```

Comment:
The amount of ETH required varies based on the network - witti does use 32ETH but the network definition can modify it (minimal uses 3.2ETH).

Reply:
Done